packages/compiler-cli/ngcc/src/packages/entry_point_manifest.ts

```diff
@@ -128,6 +128,18 @@ export class EntryPointManifest {
   }
 }
 
+/**
+ * A specialized implementation of the `EntryPointManifest` that always returns `null` rather than a
+ * set of entry-points from a manifest file.
+ *
+ * This is used if you want to ignore a manifest file that is written to disc.
```

Comment:
```suggestion
 * This is used if you want to ignore a manifest file that is written to disk.
```

Or are you holding on to portable media 💿😄